app/controllers/catalog_controller.rb

```diff
@@ -119,37 +118,37 @@ def show
-    #config.add_index_field 'language_facet', label: 'Language'
-    #config.add_index_field 'published_display', label: 'Published'
-    #config.add_index_field 'published_vern_display', label: 'Published'
-    #config.add_index_field 'lc_callnum_display', label: 'Call number'
-    config.add_index_field 'office_name_tesim', :label => 'Office'
-    config.add_index_field 'jurisdiction_tesim', :label => 'Jurisdiction'
-    config.add_index_field 'date_tesim', :label => 'Year'
-    config.add_index_field 'state_name_tesim', :label => 'State'
-    config.add_index_field 'party_affiliation_sim', :label => 'Party Affiliation'
+    # config.add_index_field 'title_display', label: 'Title'
+    # config.add_index_field 'title_vern_display', label: 'Title'
+    # config.add_index_field 'author_display', label: 'Author'
+    # config.add_index_field 'author_vern_display', label: 'Author'
+    # config.add_index_field 'format', label: 'Format'
+    # config.add_index_field 'language_facet', label: 'Language'
+    # config.add_index_field 'published_display', label: 'Published'
+    # config.add_index_field 'published_vern_display', label: 'Published'
+    # config.add_index_field 'lc_callnum_display', label: 'Call number'
+    config.add_index_field 'office_name_tesim', label: 'Office'
+    config.add_index_field 'jurisdiction_tesim', label: 'Jurisdiction'
+    config.add_index_field 'date_tesim', label: 'Year'
+    config.add_index_field 'state_name_tesim', label: 'State'
+    config.add_index_field 'party_affiliation_sim', label: 'Party Affiliation'
 
     # solr fields to be displayed in the show (single result) view
     #   The ordering of the field names is the order of the display
-    #config.add_show_field 'title_display', label: 'Title'
-    #config.add_show_field 'title_vern_display', label: 'Title'
-    #config.add_show_field 'subtitle_display', label: 'Subtitle'
-    #config.add_show_field 'subtitle_vern_display', label: 'Subtitle'
-    #config.add_show_field 'author_display', label: 'Author'
-    #config.add_show_field 'author_vern_display', label: 'Author'
-    #config.add_show_field 'format', label: 'Format'
-    #config.add_show_field 'url_fulltext_display', label: 'URL'
-    #config.add_show_field 'url_suppl_display', label: 'More Information'
-    #config.add_show_field 'language_facet', label: 'Language'
-    #config.add_show_field 'published_display', label: 'Published'
-    #config.add_show_field 'published_vern_display', label: 'Published'
-    #config.add_show_field 'lc_callnum_display', label: 'Call number'
-    #config.add_show_field 'isbn_t', label: 'ISBN'
+    # config.add_show_field 'title_display', label: 'Title'
+    # config.add_show_field 'title_vern_display', label: 'Title'
+    # config.add_show_field 'subtitle_display', label: 'Subtitle'
+    # config.add_show_field 'subtitle_vern_display', label: 'Subtitle'
+    # config.add_show_field 'author_display', label: 'Author'
```

Comment:
This may be a good time to just delete these big commented blocks? I'll leave it up to you but a lot of that code gets generated by the blacklight generator and we're never going to use it in this interface.